### DISCUSSION TITLE
Secret Scan alerts need flag/filter for unrevoked tokens

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#49086] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

I unfortunately need a way (which is really sad) to view tokens that have still not been revoked after closing the alarm.

**Comment by Timmmy-nlb** [comment]
@SayedAlAhlMohamed thank you for this hint.
But in my case it's from an enterprise view with manager not developer intention: 

I have to ensure that the developer make her job and revoke there own tokens (not my own).

That is the reason why I need this additional filter option.

**Comment by 15MariamS** [comment, marked as answer]
Hi @Timmmy-nlb, 

We now show [whether a GitHub token is active](https://github.blog/2023-01-19-remediation-made-simple-introducing-new-validity-checks-for-github-tokens/) or not in the UI. Are you looking for a way to filter the UI (and/or API) for still-active secrets? If so, it's on our roadmap (UI first!), and we'll continue to improve validation checks for supported secrets.

**Comment by Timmmy-nlb** [comment]
Hello @15MariamS ,
the filter was the thing what I need.
If is already on your post-it, I'm fine.
